aea/contracts/base.py

```diff
@@ -62,18 +61,21 @@ def configuration(self) -> ContractConfig:
         assert self._configuration is not None, "Configuration not set."
         return cast(ContractConfig, super().configuration)
 
-    @classmethod
-    @abstractmethod
-    def get_instance(
-        cls, ledger_api: LedgerApi, contract_address: Optional[str] = None
-    ) -> Any:
-        """
-        Get the instance.
-
-        :param ledger_api: the ledger api we are using.
-        :param contract_address: the contract address.
-        :return: the contract instance
-        """
+    # @classmethod
```

Comment:
Not sure if it is useful; as we have similar functionality exposed on ledger api